### DISCUSSION TITLE
App not Running after run "NPM RUN DEV"

### DISCUSSION THREAD

**Discussion by greatkhanjoy** [Shopify/shopify-app-template-node#1047] [Q&A] [answered]
Hi, I have installed this app with npm init @shopify/app@latest .
but when I run "npm run dev"  exited after "Dependencies installed" .
check this screenshot  https://prnt.sc/RNjxi9XSYYE5

**Comment by DennisKraaijeveld** [comment, marked as answer]
Have the exact same :) I downgraded to 3.10 and everything is working fine again.

**Reply by adiba178** [reply]
3.10 CLI ?

**Reply by greatkhanjoy** [reply]
Thanks . It works with 3.10.0 :)
